downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -200,6 +200,32 @@ This must be reachable from client machines.
 | *`sso_use_https`* | If Single Sign On uses https.
 
 Default = `true`
+| *`sso_console_admin_username`* | SSO admin username.
+
+Default = `admin`
+| *`sso_ssl_validate_certs`* | Set to `true` if the cert is to be validated during connection.
+
+Default = `true`
+| *`sso_custom_keystore_file`* | Customer-provided keystore for SSO.
+
+| *`sso_keystore_file_remote`* |  Set to `true` if the customer-provided keystore is on a remote node.
+
+Default = `false`
+| *`sso_keystore_name`* | Name of keystore for SSO.
+
+Default = `ansible-automation-platform`
+| *`sso_automation_platform_realm`* | The name of the realm in SSO.
+
+Default = `ansible-automation-platform`
+| *`sso_automation_platform_realm_displayname`* | Display name for the realm.
+
+Default = `Ansible Automation Platform`
+| *`sso_automation_platform_login_theme`* | Path to the theme files.
```

Comment:
If changing this var, customers need to provide their own theme files and this var needs to be path to the folder where theme files are located.